styles.css

```diff
@@ -0,0 +1,75 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-link {
+  margin: 2%;
+  color: #6070ff;
+  margin-top: 10%;
+}
+
+.logo {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.icon-links {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  margin-left: 0;
+}
+
+.icon-links li {
+  margin-right: 5%;
+}
```

Comment:
- [ ] Alignment of this section is also off as compared to the provided Figma design. Take a look 👇🏼. Consider working on it.

Implementation            |  Actual Design
:-------------------------:|:-------------------------:
![social_imp](https://user-images.githubusercontent.com/45002654/125026096-8f59c380-e08c-11eb-9693-452d5e68fb6e.PNG)  |  ![social](https://user-images.githubusercontent.com/45002654/125026116-984a9500-e08c-11eb-98b6-de5821c487c6.PNG)